<ui version="4.0" >
 <class>EntityStatistics</class>
 <widget class="QWidget" name="EntityStatistics" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EntityStatistics</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Number of States:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="numStates" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Not available</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Real states:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="numReal" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Not available</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Ghost states:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="numGhost" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Not available</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Injection States</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="numInjections" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Not available</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="numDowngraded" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Not available</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Downgraded</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="instances" >
      <column>
       <property name="text" >
        <string>Show</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>InstanceId</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Connection</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Kind</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Handler</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string># Subscribers</string>
       </property>
      </column>
     </widget>
     <widget class="QTextEdit" name="information" />
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
